smartsim/_core/_cli/validate.py

```diff
@@ -112,10 +113,11 @@ def execute(
     except Exception as e:
         logger.error(
```

Comment:
I look at these black reformats and wonder if anybody dislikes them as much as me. I'm a fan of:

```py
msg = f"Experiment failed due to the following exception:\n{e}"
logger.error(msg, exc_info=True)
```

just to avoid turning 2 lines into 4

Reply:
Hahaha a fair point, but in order to maintain the concatenation with the rest of the string and stay within the 88 char limit, we would have to result to a
```py
msg = "SmartSim failed to run a simple experiment!\n"
msg += f"Experiment failed due to the following exception:\n{e}"
logger.error(msg, exc_info=True)
```
And I hate the idea of creating a buffer and then immediately needing to create another second buffer and copy the contents of the original buffer into the new one to save ~1 line, which why I tend to prefer the black styling for long strings.